site.playbook.yml

```diff
@@ -35,10 +35,13 @@ content:
   sources:
     - url: https://github.com/cppalliance/site-docs
       start_path: user-guide
+      edit_url: '{web_url}/edit/${page-boost-branch}/{path}'
```

Comment:
I don't know where {web_url} is coming from, but the branch should always be develop.

Reply:
@alandefreitas `web_url ` holds our repo link in antora ui.